core/src/mindustry/input/Binding.java

```diff
@@ -69,6 +69,7 @@
     chat_history_prev(KeyCode.up),
     chat_history_next(KeyCode.down),
     chat_scroll(new Axis(KeyCode.scroll)),
+    chat_mode(KeyCode.tab),
```

Comment:
Tab is already used for the player list.

Reply:
1. Player list isn't displayed by `Tab` when chat is open.
2. What could be better key then? :eyes:

Reply:
Ctrl + Tab